src/redteam/plugins/base.ts

```diff
@@ -230,15 +230,38 @@ export abstract class RedteamGraderBase {
   }): ResultSuggestion[] {
     return [];
   }
+  protected handleRefusal(
+    provider: ApiProvider | undefined,
+    isRefusal: boolean | undefined,
+  ): { grade: GradingResult; rubric: string } | undefined {
+    if (isRefusal) {
+      return {
```

Comment:
Because refusals pass by default, aren't we ok without this logic? 

Would it also be nice to see the actual refusal output, which is the current behavior?

Reply:
> Because refusals pass by default, aren't we ok without this logic?

How do you suggest refactoring it? 

> Would it also be nice to see the actual refusal output, which is the current behavior?

This is what a refusal looks like in the webui with the current implementation. 

<img width="1229" alt="Screenshot 2024-10-27 at 12 38 04 AM" src="https://github.com/user-attachments/assets/9eae177b-ddc9-4f2f-8974-bfa36a828dd4">

What exactly are you proposing? That I write the llmOutput to value?